packages/web/src/ui/components/HealthChecks/APIHealth.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+
+import { HealthChecker } from "./Containers/HealthChecker";
+import { StatusPresentation } from "./Presentation/Status";
+export class APIHealth extends React.Component {
+  public render() {
+    return (
+      <HealthChecker
+        name="API Server"
+        url="/health_check"
+        component={StatusPresentation}
```

Comment:
@grimesjm do you happen to know that type `component` would be in the props?

I tried:
```
(props: IWhateverProps) => React.Component<IWhateverProps>
React.Component<IWhateverProps>
(props: IWhateverProps) => Element
```
They all have various errors